generators/cleanup.js

```diff
@@ -343,6 +343,10 @@ function cleanupOldFiles(generator) {
       generator.removeFile('.npmrc');
     }
   }
+  if (generator.isJhipsterVersionLessThan('7.8.2')) {
+    generator.removeFile(`${constants.DOCKER_DIR}realm-config/jhipster-users-0.json`);
+    generator.removeFile('realm-config/jhipster-users-0.json'); // docker-compose
```

Comment:
Should create a cleanup task here:
https://github.com/jhipster/generator-jhipster/blob/dce15044b1f74a9345447ec8181c3945eb1da113/generators/docker-compose/files.js#L29

```
  cleanup() {
      this.removeFile('realm-config/jhipster-users-0.json');
  },
```